fog/sql_recovery_db/src/test_utils.rs

```diff
@@ -36,13 +44,7 @@ impl SqlRecoveryDbTestContext {
         // database.
         let postgres_url = format!("{}/postgres", base_url);
         log::info!(&logger, "Connecting to root PG DB {}", postgres_url);
-        let conn = PgConnection::establish(&postgres_url).unwrap_or_else(|err| {
-            panic!(
-                "Cannot connect to PG database '{}': {:?}",
-                postgres_url, err
-            )
-        });
-
+        let conn = SqlRecoveryDbTestContext::establish_connection(&postgres_url);
```

Comment:
Has previous failed CI runs indicated that this is where the flakiness originated?
Just making sure in case there's some annoying edge-case where this succeeds because it doesn't do anything until we try and do something with the connection.

Reply:
Yes, in the past month about 5 CI runs failed with the "Could not connect to Posgres" message.